### PR TITLE
Interesting Taps: cleanup

### PR DESCRIPTION
**Pull request by dunn**
Also alphabetized the Homebrew taps.

### CODE REVIEW

File: share/doc/homebrew/Interesting-Taps-&-Branches.md [hunk 1]
```diff
@@ -4,91 +4,63 @@ Homebrew has the capability to add (and remove) multiple taps to your local inst
...
 
-*   [wilmoore/homebrew-home](https://github.com/wilmoore/homebrew-home)
-    - Homebrew install for those that like to Homebrew @ $HOME (i.e. ~/.homebrew).
+*   [homebrew/linuxbrew](https://github.com/Homebrew/linuxbrew): Experimental Linux version
 
-*   [nddrylliog/homebrew-mingw](https://github.com/nddrylliog/homebrew-mingw)
-    - An experimental port of Homebrew for Windows (with an MSYS/MinGW environment).
+*   [wilmoore/homebrew-home](https://github.com/wilmoore/homebrew-home): Homebrew install for those that like to Homebrew @ $HOME (i.e. ~/.homebrew).
 
+*   [nddrylliog/homebrew-mingw](https://github.com/nddrylliog/homebrew-mingw): An experimental port of Homebrew for Windows (with an MSYS/MinGW environment).
```
Comment (DomT4) [inline share/doc/homebrew/Interesting-Taps-&-Branches.md]: I'd be tempted to get rid of this one. It hasn't been updated in a *loooooong* time.

**Comment by DomT4** [comment]
LGTM the one comment aside.

File: share/doc/homebrew/Interesting-Taps-&-Branches.md (hunk 1 above)
Comment (dunn) [inline share/doc/homebrew/Interesting-Taps-&-Branches.md]: It's a cool idea though, maybe someone will see it and take it up again ¯\(シ)/¯

File: share/doc/homebrew/Interesting-Taps-&-Branches.md [hunk 2]
```diff
@@ -4,91 +4,63 @@ Homebrew has the capability to add (and remove) multiple taps to your local inst
...
-    - Need GDB or a newer Tk? System duplicates go here.
+*   [homebrew/binary](https://github.com/Homebrew/homebrew-binary): Precompiled binary formulae.
 
-*   [homebrew/versions](https://github.com/Homebrew/homebrew-versions)
-    - Need e.g. older or newer versions of Postgresql? Older versions of GCC?
+*   [homebrew/boneyard](https://github.com/Homebrew/homebrew-boneyard): Formula are not deleted, they are moved here.
 
-*   [homebrew/games](https://github.com/Homebrew/homebrew-games)
-    - Game or gaming-emulation related formulae.
+*   [homebrew/brewdler](https://github.com/Homebrew/homebrew-brewdler): A Bundler-equivalent for installing project dependencies from Homebrew.
```
Comment (xu-cheng) [inline share/doc/homebrew/Interesting-Taps-&-Branches.md]: This has been renamed to `homebrew/bundle`

Comment (dunn) [inline share/doc/homebrew/Interesting-Taps-&-Branches.md]: Oh right, thanks!

**Comment by mikemcquaid** [comment]
:+1:

**Pull request closed without merging**